### PR TITLE
[937] Enable Azure RBAC authentication to AKS

### PR DESCRIPTION
**Pull request by saliceti**
### Context

Ticket: https://trello.com/c/3m8PNLG6

The AKS clusters will be migrated to Azure RBAC and the authentication need to change.

### Changes proposed in this pull request
- Use kubelogin to enable Azure RBAC
- Remove azure credentials for terraform as they are provided by set-kubelogin-environment

### CODE REVIEW

**Comment by saliceti** [comment]
Closing for testing

**Comment by saliceti** [comment]
Closing for retesting

**Comment by saliceti** [comment]
Closing for reretesting

**Comment by saliceti** [comment]
Closing for rereretesting

File: terraform/application/config/review.tfvars.json [hunk 1]
```diff
@@ -3,6 +3,5 @@
     "namespace": "cpd-development",
     "environment": "review",
     "deploy_azure_backing_services": false,
-    "enable_postgres_ssl" : false,
-    "command": ["/bin/sh", "-c", "bundle exec rails db:environment:set RAILS_ENV=review && RAILS_ENV=review bundle exec rails db:schema:load && RAILS_ENV=review bundle exec rails db:seed && bundle exec rails server -b 0.0.0.0"]
```
Comment (cwrw) [inline terraform/application/config/review.tfvars.json:7]: where do we do this for review apps now this has been removed out of interest?

**Review by cwrw — APPROVED** [review]

File: terraform/application/config/review.tfvars.json (hunk 1 above)
Comment (saliceti) [reply terraform/application/config/review.tfvars.json:7]: I've discussed it with @slawosz , this variable is not used
I think the seeding is done here: https://github.com/DFE-Digital/teaching-school-hub-finder/blob/main/.github/actions/deploy-environment-to-aks/action.yml#L55-L60

**Review by slawosz — APPROVED** [review]

**Review by leoapost — APPROVED** [review]

**Pull request merged**